LightweightCharts/Classes/LightweightChartsModels/SeriesType.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+
+public enum SeriesType: String {
+    case line = "Line"
+    case area = "Area"
+    case candlestick = "Candlestick"
+    case bar = "Bar"
+    case histogram = "Histogram"
+    case none
+}
+
+extension SeriesType: Decodable {
+    public init(from decoder: Decoder) throws {
+        let type = try decoder.singleValueContainer().decode(String.self)
+        self = SeriesType(rawValue: type) ?? .none
+    }
+}
```

Comment:
It is enough to inherit `SeriesType` from `Decodable` protocol.
Automatic decoding will work correctly in this case.